os/arch/arm/include/armv7-a/irq.h

```diff
@@ -402,23 +406,18 @@ static inline irqstate_t up_irq_enable(void)
       : "memory"
     );
 
-  return cpsr;
+	return cpsr;
 }
 
 /* Restore saved IRQ & FIQ state */
 
 static inline void irqrestore(irqstate_t flags)
 {
-  __asm__ __volatile__
-    (
-      "msr    cpsr_c, %0"
-      :
-      : "r" (flags)
-      : "memory"
-    );
+	__asm__ __volatile__("msr    cpsr_c, %0"::"r"(flags)
+						 :"memory");
```

Comment:
Sorry but, this looks not good.
Even we must use a tab for indentation, we don't need to merge all separated lines.
Let's keep separate lines for each command
```
(tab)__asm__ __volatile__(
(tab)(tab)"msr  cpsr_c, %0"
(tab)(tab):
(tab)(tab): "r" (...)
....
..
```

Reply:
same as other __asm__ codes

Reply:
Ok, I will change as other asm code.